compose/production/nginx/nginx.conf

```diff
@@ -0,0 +1,31 @@
+# user  nginx;  # disabled to avoid a warning since we're booting up the container as nginx, rather than as root
+worker_processes  1;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
+
+    #gzip  on;
```

Comment:
Why off ?

Reply:
This was what the config was by default. The default official nginx image has this line commented out.

Reply:
Maybe we can activate it to save some bandwidth

Reply:
Maybe, but bandwidth for nginx should be completely trivial - the only hits to the nginx will be when people access individual SGF files manually in the browser, because networks downloads are hosted on cloud bucket.